satellite/satellitedb/orders.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/pb"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+type ordersDB struct {
+	db *dbx.DB
+}
+
+// SaveInlineOrder saves inline order
+func (db *ordersDB) SaveInlineOrder(ctx context.Context, bucketID []byte) error {
+	return nil
+}
+
+// SaveRemoteOrder saves remote order
+func (db *ordersDB) SaveRemoteOrder(ctx context.Context, bucketID []byte, orderLimits []*pb.OrderLimit2) error {
+	if len(orderLimits) == 0 {
+		return nil
+	}
+
+	expires, err := ptypes.Timestamp(orderLimits[0].OrderExpiration)
+	if err != nil {
+		return err
+	}
+
+	tx, err := db.db.Open(ctx)
+	if err != nil {
+		return err
+	}
+
+	serialNumber := orderLimits[0].SerialNumber
+
+	_, err = tx.Create_SerialNumber(
```

Comment:
this method accepts a list of `OrderLimit2`, but why is only 1 `serial_number` created? I thought we would want to list through the list of `orderLimits` and create a serial_number for each orderLimit in the list

Reply:
I'm not sure if stay as is at the end of changes. In this PR serial number is generated per uplink request. Main goal was to avoid storing one serial number for one storage node (up to 95 per request) and save DB space. Thats why we have two tables: `serial_number` and `used_serials` (serial_number|storagenode_id). Serial is added to `serial_number` and later when OrderLimit is received we are filling `used_serials`.